### ISSUE TITLE
Selecting lots of roads makes the window infinitely long.

### ISSUE THREAD

**Issue by michellegrace** [CEMPD/ctools#88]
When you add a roadway, then another, and another, etc...... 
![screen shot 2015-02-06 at 3 33 57 pm](https://cloud.githubusercontent.com/assets/10077162/6087388/bbeb66cc-ae18-11e4-9882-4f8ab2998074.png)

Eventually this window takes over the entire screen. Can you make this have multiple pages like the modify roads window?
![screen shot 2015-02-06 at 3 57 16 pm](https://cloud.githubusercontent.com/assets/10077162/6087406/f230ab8e-ae18-11e4-8a98-ab1f96cc5039.png)

**Comment by michellegrace** [comment]
Brian also mentioned this problem in his email on 4/23/15: Well, in a scenario that I ran for the students, I had to select quite a few rail segments and then make a wholesale change. There were so many segments, that the Apply button was hidden below the Clear Selection button. Since I had a ton of road, terminals, etc. selected, I couldn't clear the selections. So, I had to decrease my window size until the Apply button appeared again. I was able to run all of the scenarios successfully then, and everything was cool, since I checked the Keep Map Data button (which should read more like keep current receptor set or keep current resolution).
